src/noise-plethora/LICENSE.md

```diff
@@ -0,0 +1,2 @@
+Plugins in `./plugins` have been derived from source files from https://github.com/Befaco/Noise_plethora/ (prefix `P_*.hpp`) which were originally licensed under CC-BY-SA v4 (see https://github.com/Befaco/Noise_plethora/blob/master/README.md). This license is [one-way compatible with GPL-v3-only](https://creativecommons.org/share-your-work/licensing-considerations/compatible-licenses), which the Befaco VCV plugins are licensed under.
+
```

Comment:
I should probably add that the modified Teensy codes are MIT licensed.

Reply:
Added

Reply:
Are Befaco and Teensy licenses included in the binary package? If not, add them to `LICENSE-dist.txt` or something and add that file to `DISTRIBUTABLES += ...`

Reply:
Will do. Befaco Noise Plethora sources are now GPL-v3-or-later (same as this repo), does that still needed to be added?